en/config/actuators.md

```diff
@@ -302,7 +302,7 @@ The _Actuator Outputs_ section is used to assign motors, control surface servos,
 
 ![Actuator Outputs - Multicopter diagram](../../assets/config/actuators/qgc_actuators_mc_outputs.png)
 
-Separate tabs are displayed for each output bus supported by the connected flight controller: PWM AUX (IO Board output), PWM MAIN (FMU Board output), UAVCAN.
+Separate tabs are displayed for each output bus supported by the connected flight controller: PWM MAIN (I/O Board output), PWM AUX (FMU Board output), UAVCAN.
```

Comment:
I see, thank you for the clarification. This might be useful to mention in the docs, too.

I have some vehicles with a PixHawk 4 and some with the [mRo Control Zero H7 OEM](https://mrobotics.io/docs/mro-control-zero-oem-h7/). The mRo has no IO processor, and its outputs are as far as I can tell all MAIN, which would then be from the FMU processor. At least, using control allocation, and setting the motors to MAIN outputs, flies as expected. Does this sound correct?

Reply:
Yes, this is correct.

The information is in the reference design document you already linked to🙂. I knew it was around somewhere but I didn't realise it was in that page until I looked carefully

Reply:
Indeed, thanks again.